### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by wfcd-bot-boi**

### CODE REVIEW

File: src/lang/ko.json [hunk 1]
```diff
@@ -113,6 +113,7 @@
   },
   "conclave": {
     "header": "Conclave",
+    "weekly": "Weekly",
```
Comment (EricSihaoLin) [inline src/lang/ko.json:116]: Codacy Issue found: [Format Error: unexpected "Weekly\",\n    \"weekly\": \""](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5564722)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5564722&bid=18249000)

File: src/lang/ru.json [hunk 2]
```diff
@@ -1,115 +1,123 @@
...
-    "weekly": "Weekly",
-    "elite": "Elite Weekly",
-    "daily": "Daily"
+    "header": "Ночная Волна",
+    "weekly": "Еженедельные",
+    "elite": "Еженедельные Элитные",
+    "daily": "Ежедневные"
+  },
+  "conclave": {
+    "header": "Конклав",
```
Comment (EricSihaoLin) [inline src/lang/ru.json:115]: Codacy Issue found: [Format Error: unexpected "Конклав\",\n    \"weekly\": \"Еженедельно\",\n    \"teamannihilation\": \"Командное истребление\",\n    \"annihilation\": \"Истребление\",\n    \"lunaro\": \"Лунаро\",\n    \"cephaloncapture\": \"Захват Цефалона\"\n  },\n  \"conclave\": {\n    \"header\": \""](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5564722)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5564722&bid=18249000)

File: src/lang/ru.json (hunk 2 above)
Comment (EricSihaoLin) [inline src/lang/ru.json:115]: Codacy Issue found: [Format Error: unexpected "Конклав\",\n    \"weekly\": \"Еженедельно\",\n    \"teamannihilation\": \"Командное истребление\",\n    \"annihilation\": \"Истребление\",\n    \"lunaro\": \"Лунаро\",\n    \"cephaloncapture\": \"Захват Цефалона\"\n  },\n  \"conclave\": {\n    \"header\": \""](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5564722)

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**